### PR TITLE
Initial commit for Terracotta Server OSS helm chart

### PR DESCRIPTION
**Pull request by anthonydahanne**
Signed-off-by: Anthony Dahanne <anthony.dahanne@gmail.com>

#### What this PR does / why we need it:
Add the Terracotta Server OSS to stable helm charts; so that it can be used a clustered backend for Ehcache3 Java caching library users

#### Which issue this PR fixes
none

#### Special notes for your reviewer:
It's an initial support; things like image pull secrets or RBAC accounts are not considered yet; I plan to add them at a later time.
I believe this initial chart already brings a lot of value for existing [Terracotta Server OSS](http://www.terracotta.org/) users 
Thank you

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

**Comment by anthonydahanne** [comment]
just added missing new line for Chart.yaml

File: stable/terracotta/templates/service.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  name: {{ template "terracotta.fullname" . }}
+  labels:
+    app: {{ template "terracotta.name" . }}
+    chart: {{ template "terracotta.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    # see https://github.com/kubernetes/kubernetes/issues/39363 , to have dns entries available immediately
+    service.alpha.kubernetes.io/tolerate-unready-endpoints: "{{ .Values.tolerateUnreadyEndpoints }}"
+
```
Comment (davidkarlsen) [inline stable/terracotta/templates/service.yaml]: remove whitespace to be consistent in formatting

File: stable/terracotta/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+| `image.repository`                         | Terracotta Image name                                                                                          | `terracotta/terracotta-server-oss`                   |
+| `image.tag`                                | Terracotta Image tag                                                                                           | `{VERSION}`                                          |
+| `image.pullPolicy`                         | Image pull policy                                                                                              | `Always`                                             |
+| `replicaCount`                             | Number of Terracotta members                                                                                   | 2                                                    |
+| `offheaps`                                 | Offheap resource defintions                                                                                    | offheap-1, 512, MB and offheap-2, 256, MB            |
```
Comment (davidkarlsen) [inline stable/terracotta/README.md]: the values should be a list here too to make it more apparent

**Review by davidkarlsen — COMMENTED** [review]
nice one - a few small comments

File: stable/terracotta/README.md (hunk 2 above)
Comment (davidkarlsen) [reply stable/terracotta/README.md]: Also please add rbac now as it's one of the requirements: https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md - shouldn't take too much effort

File: stable/terracotta/templates/service.yaml (hunk 1 above)
Comment (anthonydahanne) [reply stable/terracotta/templates/service.yaml]: ok, removed empty line - coming in next commmit

File: stable/terracotta/README.md (hunk 2 above)
Comment (anthonydahanne) [reply stable/terracotta/README.md]: ok for the list.
I'll add RBAC too
thanks for reviewing, new commits coming up soon

Comment (anthonydahanne) [reply stable/terracotta/README.md]: @davidkarlsen I have pushed the fixes; please let me know if you find any issues ! Thanks again !

**Comment by anthonydahanne** [comment]
thank you for reviewing @davidkarlsen  !

**Pull request merged**